plugins/include/amxconst.inc

```diff
@@ -55,10 +55,45 @@ public stock const NULL_STRING[1];
 public stock const Float:NULL_VECTOR[3];
 
 /**
- * The maximum buffer size required to store a clients name.
+ * The maximum buffer size required to store a client's name.
  */
 #define MAX_NAME_LENGTH 32
 
+/**
+ * The maximum buffer size required to store a client's IP address without a port.
+ */
+#define MAX_IP_LENGTH 16
+
+/**
+ * The maximum buffer size required to store a client's IP address with a port.
+ */
+#define MAX_IP_WITH_PORT_LENGTH 22
+
+/**
+ * The maximum buffer size required to store a client's AuthID.
+ */
+#define MAX_AUTHID_LENGTH 64
+
+/**
+ * The maximum buffer size required to store a resource path.
+ */
+#define MAX_RESOURCE_PATH_LENGTH 64
+
+/**
+ * The maximum buffer size that can be displayed in a MOTD.
+ */
+#define MAX_MOTD_LENGTH 1536
```

Comment:
Oh, sorry, looks like it is not the same for different games too:
https://github.com/ValveSoftware/halflife/blob/5d761709a31ce1e71488f2668321de05f791b405/dmc/cl_dll/hud.h#L56
https://github.com/ValveSoftware/halflife/blob/5d761709a31ce1e71488f2668321de05f791b405/cl_dll/hud.h#L61

Reply:
but we don't know current code base.
hl now support color menus. what else has changed in current builds?

Reply:
@Mistrick that repository contains colored menus support.